### ISSUE TITLE
Method to edit keybinds for changing frequency?

### ISSUE THREAD

**Issue by IsuzeAwase** [Larkooo/FrequencyWalkie#7]
A couple other popular QoL mods overlap with the new default keybinds for this mod (R and F.) Could a more accessible method to edit the change frequency keybinds be added? 

Making it editable in something like the in-game keybinds menu _(using [InputUtils](https://thunderstore.io/c/lethal-company/p/Rune580/LethalCompany_InputUtils/))_ might be the best option to implement, and could make the mod more easily usable for people with modpacks or other keybind preferences; even if it creates a dependency. _(Should be easier than trying to make it editable using the thunderstore/r2modman config editor afaik.)_

**Comment by Larkooo** [comment]
Yes, being able to change the keybinds for changing frequencies from the in-game settings menu was already in my todo list. I just didn't have the time to implement it. 

One potentially simple but naive solution would be to implement those keybindings to be apart of the `PlayerActions` class which handles all of the binded player actions but I'm quite unsure about its interference with other mods that do the same thing or use `InputUtils`. 

InputUtils does seem like a good solution even though it adds a dependency to the mod.
